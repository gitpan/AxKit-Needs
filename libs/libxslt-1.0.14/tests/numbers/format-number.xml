<functions>
  <pi>3.1415</pi>
  <negpi>-3.1415</negpi>
</functions>
