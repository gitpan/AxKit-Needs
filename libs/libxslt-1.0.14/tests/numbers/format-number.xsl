<xsl:stylesheet version="1.0"
	      xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

<xsl:output
 method="xml"
 indent="yes"
 encoding="iso-8859-1"
/>

<xsl:decimal-format
 name = "special"
 decimal-separator = "*"
/>

<xsl:template match="functions">
 <pi>
  one <xsl:value-of select="format-number(pi, 'prefix#,#,###.##suffix')"/>
  two <xsl:value-of select="format-number(negpi, '_#,#,###.##_')"/>
  three <xsl:value-of select="format-number(negpi, '_#,#,000.000##_')"/>
  four <xsl:value-of select="format-number(negpi, '_#.#_;_(#.#)_')"/>
  five <xsl:value-of select="format-number(pi, 'prefix#,#,###*##suffix','special')"/>
 </pi>
</xsl:template>

</xsl:stylesheet>
