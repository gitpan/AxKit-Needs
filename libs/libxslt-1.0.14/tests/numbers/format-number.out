<?xml version="1.0" encoding="iso-8859-1"?>
<pi>
  one prefix3.14suffix
  two -_3.14_
  three -_003.1415_
  four _(3.1)_
  five prefix3*14suffix</pi>
